atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -331,6 +333,12 @@ public static TransactionManager createInMemory(Set<Schema> schemas) {
     @JsonIgnore
     @Value.Derived
     public TransactionManager serializable() {
+        return lockWatch();
+    }
+
+    @JsonIgnore
+    @Value.Derived
+    public LockWatchTransactionManager lockWatch() {
```

Comment:
yeah, this change can't happen - breaks too much binary backcompat internally

Reply:
Removed